app/controllers/teams_controller.rb

```diff
@@ -64,15 +64,19 @@ def team
   end
 
   def team_params
-    params.require(:team).permit(:name, :agency_id, :description, user_ids: [])
+    params.require(:team).permit(:name, :agency_id, :description)
+  end
+
+  def user_params
+    params[:user_ids]&.split(' ') || []
```

Comment:
this is necessary because the create path also uses `update_params_with_current_user ` another option would be to change that code path; i wasn't sure which was preferred